src/core/particle_data.cpp

```diff
@@ -833,26 +833,17 @@ void prefetch_particle_data(std::vector<int> ids) {
                            std::make_move_iterator(parts.begin()));
 }
 
-int place_particle(int part, double p[3]) {
-  int retcode = ES_PART_OK;
+int place_particle(int part, const double *p_) {
```

Comment:
why not ```Utils::Vector3d```? and why that name?

Reply:
Changing the type would require updating all the call sites, which is out of scope.

Reply:
As a token of good will I changed some other signatures ^^